### ISSUE TITLE
Simplified route view

### ISSUE THREAD

**Issue by Mygod** [WatWowMap/ReactMap#953]
Current route view has many issues.

1. It is impossible to read on ReactMap unless it is the only thing enabled.
2. If there are multiple routes coming out of the same POI, it is impossible to open the popup for all but the first route.
3. The current system of having two colored flags does nothing but creating more visual noise.
4. Even worse, the two flags overlap each other, making it very hard to tap on especially on mobile, even if you zoom all the way in. This is an accessibility issue since [every tappable element should have at least 48dp x 48dp hit region or something like that](https://support.google.com/accessibility/android/answer/7101858?hl=en).

I suggest to add a simplified view to make the map less cluttered. Maybe even replacing the old system outright.

1. Since routes must start/end at a POI (stop or gym), by default the only thing displayed on a map should be a 🚩 icon overlayed onto the starting point or the ending point too if it is reversible, indicating that there are at least one routes starting from this POI. 
2. When a POI is clicked on, all the routes (paths) starting from that POI will be displayed and the user can click on one of the paths to view details about the route.
3. Ideally the route should be displayed even if the map is being dragged around. In fact, this should be applied to every other element as well: the popup should not be closed just because the associated object was dragged out of the map view.

Fortunately we can just disable routes for now and use Campfire instead. :)

**Comment by Fabio1988** [comment]
I totally agree, tried to use routes last week, impossible to find a proper route on my way home, filtering on starting-point isn't a bad idea, would help a lot :)

Can't follow any route on this view: 

![Image](https://github.com/user-attachments/assets/a115b559-b1f7-48ab-b690-7b48dca4c295)

**Comment by Mygod** [comment]
Disabling routes by default for now. (https://github.com/WatWowMap/ReactMap/commit/7e8ed3bc6f726fc75e7cadf5bca60ab7ad211e3f)
